utils/S3UtilsMongoClient.js

```diff
@@ -261,6 +357,19 @@ class S3UtilsMongoClient extends MongoClientInterface {
             const retResult = this._handleResults(collRes, isVer);
             retResult.stalled = stalledCount;
 
+            if (inflightsPreScan > 0 && retResult && retResult.dataMetrics) {
+                Object.keys(retResult.dataMetrics.bucket).forEach(key => {
+                    retResult.dataMetrics.bucket[key] = {
+                        ...retResult.dataMetrics.bucket[key],
+                        usedCapacity: {
+                            ...retResult.dataMetrics.bucket[key].usedCapacity,
+                            _inflightsPreScan: inflightsPreScan,
+                        },
+                    };
+                    retResult.dataMetrics.bucket[key].accountOwnerID = accountBucket;
```

Comment:
nit: kind of weird to replace the whole object for one field (`usedCapacity`) and update the other field separately...

either:
```suggestion
                    retResult.dataMetrics.bucket[key].usedCapacity: {
                        ...retResult.dataMetrics.bucket[key].usedCapacity,
                        _inflightsPreScan: inflightsPreScan,
                    };
                    retResult.dataMetrics.bucket[key].accountOwnerID = accountBucket;
```
or
```suggestion
                    retResult.dataMetrics.bucket[key] = {
                        ...retResult.dataMetrics.bucket[key],
                        usedCapacity: {
                            ...retResult.dataMetrics.bucket[key].usedCapacity,
                            _inflightsPreScan: inflightsPreScan,
                        },
                        accountOwnerID = accountBucket;
                    };
```

Reply:
I chose the first solution, PR was updated